ReactNativeClient/lib/components/shared/config-shared.js

```diff
@@ -156,6 +156,12 @@ shared.settingsSections = createSelector(
 			isScreen: true,
 		});
 
+		output.push({
+			name: 'keymap',
+			metadatas: [],
+			isScreen: true,
+		});
```

Comment:
As this is shared with mobile, please check that mobile still works.

Reply:
I've checked it and it works without any issue..